src/Views/Search/Book.js

```diff
@@ -0,0 +1,49 @@
+import { useHistory } from 'react-router-dom'
+
+const Book = ({ data }) => {
+    const {
+      name,
+      author,
+      genres,
+      year,
+      publisher,
+      description,
+      ownerId,
+    } = data;
```

Comment:
`Comment 1, Part 1/2` - I see that you are using there destructurizing of an object? Or something called similarly 😅  If you add what I told in `Comment 1, Part 2/2` then you could  use only one line for those thing, like that:

`const Book = ({name, author, ..., ownerId}) => { `
destructurize where it is possible!